src/Entity.php

```diff
@@ -7,12 +7,19 @@
 class Entity extends Resource implements ArrayableInterface
 {
     protected $attributes = array();
-
+    private $additionHeader;
+ /**
+     * Create method 
+     *
+     * @param array $attributes 
+     *
+     * 
+     */
     protected function create($attributes = null)
     {
         $entityUrl = $this->getEntityUrl();
-
-        return $this->request('POST', $entityUrl, $attributes);
+        
+        return $this->request('POST', $entityUrl, $attributes, $this->additionHeader);
```

Comment:
additional header is passed here. cannot see the function param in request function